apps/edxec/templates/services/app/job_01_db_migrate.yml.j2

```diff
@@ -0,0 +1,77 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: edxec-dbmigrate-{{ job_stamp }}
+  namespace: "{{ project_name }}"
+  labels:
+    app: edxec
+    service: edxec
+    version: "{{ edxec_image_tag }}"
+    job_stamp: "{{ job_stamp }}"
+    deployment_stamp: "{{ deployment_stamp }}"
+spec:
+  template:
+    metadata:
+      name: edxec-dbmigrate-{{ job_stamp }}
+      labels:
+        app: edxec
+        service: edxec
+        version: "{{ edxec_image_tag }}"
+        job_stamp: "{{ job_stamp }}"
+        deployment_stamp: "{{ deployment_stamp }}"
+    spec:
+      containers:
+        - name: edxec-dbmigrate-{{ job_stamp }}
+          image: "{{ edxec_image_name }}:{{ edxec_image_tag }}"
+          imagePullPolicy: Always
+          # Describe environment explicitly
+          env:
+            - name: ECOMMERCE_CFG
+              value: /usr/local/etc/ecommerce/local.yml
+            - name: DJANGO_SETTINGS_MODULE
+              value: ecommerce.settings.production
+          command:
+            - "bash"
+            - "-c"
+            - python manage.py migrate
+          volumeMounts:
+            - name: edxec-local
+              mountPath: /usr/local/etc/ecommerce
+            - name: edxec-v-media
+              mountPath: /app/media
+            - name: edxec-v-static
```

Comment:
You shouldn't need to mount statics to run db_migrate?

Reply:
:pray: